src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/notifications/service/EmailService.java

```diff
@@ -37,6 +38,7 @@ public class EmailService {
 
     private static final String CONTESTED = "contested";
     private static final String CONSENTED = "consented";
+    private static final String FR_ASSIGNED_TO_JUDGE = "FR_ASSIGNED_TO_JUDGE";
```

Comment:
Is this needed? FR_ASSIGNED_TO_JUDGE is being imported above.

Reply:
Removed import to align with current pattern in EmailService